### ISSUE TITLE
punctuation 

### ISSUE THREAD

**Issue by MCM-Mike** [Sykh/CoinWatch#2]
can you explain if the BTC price needs to be written in 
e.g. 
German   0,00000055  (comma)
English    0.00000055 (point)

**Comment by Sykh** [comment]
Thats correct, you use the comma to seperate the decimals and a point for thousend, one million is like this
1.000.000,00
